### PR TITLE
Command-line report timeout parameter

### PR DESCRIPTION
**Pull request by ceallen**
Adds additional notebooker-cli command-line parameter to configure report timeouts.

Keeps default of 60.

Also fixed documentation that still referred to environment variable setting for python template directories.

### CODE REVIEW

File: notebooker/settings.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ class BaseConfig:
     # Value used in the from header of emails sent by notebooker if the user doesn't pass one when running a notebook
     DEFAULT_MAILFROM: str = DEFAULT_MAILFROM_ADDRESS
 
+    RUNNING_TIMEOUT: int = 60
```
Comment (jonbannister) [inline notebooker/settings.py:36]: set this to DEFAULT_RUNNING_TIMEOUT

File: docs/templates.rst [hunk 2]
```diff
@@ -27,10 +27,10 @@ To use a git repository as a notebook templates repository, you simply need to c
 :code:`notebook_requirements.txt`, containing extra package requirements to be
 installed, should be added to that folder.
 
-For Notebooker to use a your checked-out repository, set two environment variables:
+For Notebooker to use a your checked-out repository, set two command-line parameters:
 
-* Set :code:`PY_TEMPLATE_BASE_DIR` to the checked-out repository
-* Set :code:`PY_TEMPLATE_SUBDIR` to the subdirectory within your git repo which contains the templates
+* Set :code:`--py-template-base-dir` to the checked-out repository
+* Set :code:`--py-template-subdir` to the subdirectory within your git repo which contains the templates
```
Comment (jonbannister) [inline docs/templates.rst:33]: Thank you!

**Comment by jonbannister** [comment]
Add to the CHANGELOG please :)

**Comment by ceallen** [comment]
Added to CHANGELOG, and removed the duplicate constant from settings.py.

**Pull request merged**